ios/FluentUI/Popup Menu/PopupMenuController.swift

```diff
@@ -194,6 +209,21 @@ open class PopupMenuController: DrawerController {
         view.isHidden = true
         return view
     }()
+    private lazy var searchView: UIView = {
+        let view = UIStackView()
+        view.axis = .vertical
+        view.addArrangedSubview(searchBar)
+        view.isHidden = searchBar.isHidden
+        view.isLayoutMarginsRelativeArrangement = true
+        view.layoutMargins = UIEdgeInsets(top: 16, left: 16, bottom: 12, right: 16)
```

Comment:
should it be using our global tokens?

Reply:
Oh good catch forgot to update that. Thanks!